cli/cmd/stat.go

```diff
@@ -89,7 +89,8 @@ If no resource name is specified, displays stats about all resources of the spec
 
 		output, err := requestStatsFromAPI(client, req)
 		if err != nil {
-			return err
+			fmt.Fprintln(os.Stderr, err.Error())
+			os.Exit(1)
```

Comment:
what do you think about the `return fmt.Errorf(...` pattern above this line? it also returns with exit code 1. i have a slight preference for that, as it's one-line, and `return` feels less severe than `os.Exit`, but i'm ok with either approach. i do think making them all the same would be good.

Reply:
If the `RunE` func returns an error, then cobra will print the full command usage message after printing the error. My goal in making this change was to print the API error and exit, without printing the full usage message. But you're right that this is now inconsistent with the other error cases in this func. I'll revert this.